salt/orchestrate/aws/mitx_elb.sls

```diff
@@ -43,9 +43,11 @@ create_elb_for_edx_{{ purpose_name }}:
           timeout: 300
     - cnames:
         {% for domain_key, domain in purpose.domains.items()  %}
+        {% if not (edx_type == 'live' and domain_key == 'studio') %}
```

Comment:
Isn't `domain_key` in this case `cms` and not `studio`?

Reply:
Yes